### DISCUSSION TITLE
Sauvegarde et restauration

### DISCUSSION THREAD

**Discussion by Signormandiedev** [EsriPS/arcgis-assistant-feedback#68] [Q&A]
Bonjour,
J’utilise Arcgis assistant pour faire des sauvegardes de mes couches, de mes Web Map et de mes différentes applications.

J’ai remarqué que lorsque je devais restaurer une couche ou une Web Map, les applications qui en dépendent ne fonctionnent plus malgré la restauration de ces couches ou WebMap .

Par contre, si je n’ai qu’une application à restaurer (les couches ou la Web Map n’ont pas besoin d’être restaurées) la sauvegarde fonctionne bien.

Est-ce là un fonctionnement normalement d’Argis assistant, ou y a-t-il quelque chose qui m’échappe dans ma procédure de sauvegarde ou de restauration ?

Par avance merci de votre aide

**Comment by jpeterson** [comment]
Hello @Signormandiedev,

Can you describe your process for backing up and restoring web maps and apps? Since Assistant only modifies the JSON representing your web maps, everything working will depend on the items (layers, basemaps, etc.) referenced by your web app still being available (published, shared properly, etc.)

Thanks!
